velox/exec/WindowPartition.h

```diff
@@ -23,6 +23,17 @@
 /// TODO: This implementation will be revised for Spill to disk semantics.
 
 namespace facebook::velox::exec {
+
```

Comment:
I feel this is a function property and should be in WIndowFunction.h

Reply:
@aditi-pandit Yes. Move this property into WindowFunction.h